test/utils/RandomTest.cpp

```diff
@@ -47,4 +49,117 @@ TEST(Random, gaussian) {
     }
     EXPECT_GE(successes, 90);  // Leave some width for 95% confidence interval.
     // The actual value is 94, which is a good indication everything works as intended.
-}
\ No newline at end of file
+}
+
+TEST(Random, simpleIntRange) {
+    constexpr int RANGE = 20;
+    for (int i = 0; i < 50; i++) {
+        int base = SimpleRandom::getInt();
+        int low = base - RANGE;
+        int high = base + RANGE;
+
+        for (int j = 0; j < 100; j++) {
+            int test = SimpleRandom::getInt(low, high);
+            EXPECT_GE(test, low);
+            EXPECT_LE(test, high);
+        }
+    }
+}
+
+TEST(Random, simpleInt) {
```

Comment:
This test with fail with a probability of 7.8886091e-31 even when getInt() works as expected. If it does fail due to chance, Floris has to trek a bak imo